tools/osv-linter/internal/helpers/utility.go

```diff
@@ -0,0 +1,70 @@
+package helpers
```

Comment:
similar comment here re package naming: we should move this to another package.

Reply:
Done in 28bfae6